meilisearch-auth/src/store.rs

```diff
@@ -242,9 +243,12 @@ impl<'a> milli::heed::BytesEncode<'a> for KeyIdActionCodec {
 }
 
 pub fn generate_key_as_base64(uid: &[u8], master_key: &[u8]) -> String {
-    let key = [uid, master_key].concat();
-    let sha = Sha256::digest(&key);
-    base64::encode_config(sha, base64::URL_SAFE_NO_PAD)
+    let master_key_sha = Sha256::digest(master_key);
+    let mut mac = Hmac::<Sha256>::new_from_slice(master_key).unwrap();
```

Comment:
this can be unwrapped safely because `Hmac::new_from_slice` never returns an error:
https://docs.rs/hmac/0.12.1/src/hmac/optim.rs.html#152-174